test/models/dto/AddressDtoSpec.scala

```diff
@@ -318,11 +316,9 @@ class AddressDtoSpec extends BaseSpec {
 
       AddressDto.ukForm.bind(formData).fold(
         formWithErrors => {
-          formWithErrors.errors.length shouldBe 1
-          formWithErrors.errors.head.message shouldBe "error.line4_required"
         },
         success => {
-          fail("Form should give an error")
+          success shouldBe AddressDto("Line 1", "Line 2", Option("Line 3"), None, Option("Line 5"), Some("AA1 1AA"), None, None)
```

Comment:
```suggestion
          success shouldBe AddressDto("Line 1", "Line 2", Some("Line 3"), None, Option("Line 5"), Some("AA1 1AA"), None, None)
```

Reply:
Added fail

Reply:
Thanks but this comment was about using `Some` instead of `Option`